jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -250,15 +250,20 @@ def mos_image_parser(app, data_obj, data_labels=None):
         the mosviz table.
     data_labels : str, optional
         The label applied to the glue data component.
+    share_image : int, optional
+        If 0, images are treated as applying to individual spectra. If non-zero,
+        a single image will be shared by multiple spectra.
     """
-    # Parse and load the 2d images. `CCData` objects require a unit be defined
-    #  in the fits header, however, if none is provided, use a fallback and
-    #  raise an error.
 
     if data_obj is None:
         return
 
     def _parse_as_image(path):
+        """
+        Parse and load a 2D image. `CCData` objects require a unit be defined
+        in the fits header - if none is provided, use a fallback and
+        raise an error.
+        """
         with fits.open(path) as hdulist:
             if 'BUNIT' not in hdulist[0].header:
                 logging.warning("No 'BUNIT' defined in the header, using 'Jy'.")
```

Comment:
That is a dangerous assumption, but perhaps out of scope.

Reply:
I agree 100%, but yes it's out of scope.